tests/test_async_summarization.py

```diff
@@ -141,7 +141,24 @@ def test_get_transcript_summary_text(self, mock_session, make_mock_response):
             headers=self.hdr_fixture(client, {'Accept': 'text/plain'})
         )
 
-    def test_get_transcript_summary_json_paragraph(self, mock_session, make_mock_response):
+    def test_get_transcript_summary_object(self, mock_session, make_mock_response):
```

Comment:
It looks like we are missing a test for json method now, no?

Reply:
Both are there, it's just a weird diff view